features/support/electron.js

```diff
@@ -17,6 +21,24 @@ const printMainProcessLogs = () => (
     })
 );
 
+const getFile = (testName) => {
```

Comment:
Let's rename this to `generateFilePath` since that's what's happening 😉

Reply:
✅